chore(Alert): Allowed for non header elements to be set at the Alert title (#8518)
patternfly/patternfly-react: packages/react-core/src/components/Alert/examples/Alert.md

* As demonstrated in the 3rd and 4th variations below, use the `actionClose` property to add an `<AlertActionCloseButton>` component, which can be used to manage and customize alert dismissals. `actionClose` can be used with or without the presence of `actionLinks`.

* As demonstrated in the 5th and 6th variations below, use the `component` property to set the element for an alert title. 

>   * If there is no description below the alert title, then the `component` prop could be set to a non-heading element such as a `span` or `div`. 

Reviewer: Nit, just to make it more of a recommendation:

```suggestion
  * If the `description` prop is not passed in, then the `component` prop should be set to a non-heading element such as a `span` or `div`. 
```

We should also open up a followup issue for org to update the a11y docs

Author: done! https://github.com/patternfly/patternfly-org/issues/3336
